tests/distributed/test_join_local_from_remote/flow.yml

```diff
@@ -3,6 +3,7 @@ version: '1'
 with:
   port_expose: 45678
   show_exc_info: true
+  host: flow
```

Comment:
Is using 'flow' as `host` the way to specify it as localhost?

Reply:
No, this is coupled with the `docker-compose` naming scheme of docker pods.